src/pages/token/rewards/TradingRewardsSummaryPanel.tsx

```diff
@@ -15,23 +16,26 @@ import { Details } from '@/components/Details';
 import { Output, OutputType } from '@/components/Output';
 import { Panel } from '@/components/Panel';
 
+import { calculateCanViewAccount } from '@/state/accountCalculators';
 import { getHistoricalTradingRewardsForCurrentWeek } from '@/state/accountSelectors';
 
 import abacusStateManager from '@/lib/abacus';
 
 export const TradingRewardsSummaryPanel = () => {
   const stringGetter = useStringGetter();
   const { chainTokenLabel } = useTokenConfigs();
+  const canViewAccount = useSelector(calculateCanViewAccount);
   const currentWeekTradingReward = useSelector(
     getHistoricalTradingRewardsForCurrentWeek,
     shallowEqual
   );
 
   useEffect(() => {
-    abacusStateManager.refreshHistoricalTradingRewards();
-  }, []);
+    // Initialize weekly data for currentWeekTradingReward
+    abacusStateManager.setHistoricalTradingRewardPeriod(HistoricalTradingRewardsPeriod.WEEKLY);
```

Comment:
do we actually need this? i thinkkkk it's by default WEEKLY in abacus (we wanted to show weekly reward history in table by default) / for the summary

Reply:
this kinda conflicts with `abacusStateManager.setHistoricalTradingRewardPeriod(HistoricalTradingRewardsPeriod.DAILY);`  is the idea to get both data? but wonder which one will be selected in the UI